### PR TITLE
Include jupyter_notebook in doc

### PR DESCRIPTION
**Pull request by Quasimewdo**
Examples are essential for users to understand the functionnalities 
of API. This patch adds basic and advanced data science examples
of using GreenplumPython.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -48,4 +48,6 @@ skip_install = true
 commands =
     # for doc generate we need to install this moudle that can `import greenplumpython`
     pip install .
+    sudo apt install pandoc
```
[the remark this answers is not in the document]
Comment (Quasimewdo) [reply tox.ini:51]: It seems there are different ways to install [Pandoc](https://pandoc.org/installing.html) regarding the system

**Comment by yihong0618** [comment]
@Quasimewdo mv the install to `GitHub Actions`

**Comment by Quasimewdo** [comment]
> @Quasimewdo mv the install to `GitHub Actions`

Thx. If everything is done, can you add a +1 pls?

**Comment by Quasimewdo** [comment]
> It would be easier to review if we can commit the source of documents as plain text.
> 
> To do that, maybe we can use something like Jupytext https://jupytext.readthedocs.io/en/latest/

Done

**Review by yihong0618 — APPROVED** [review]

**Pull request merged**